sdcm/cluster.py

```diff
@@ -453,22 +454,35 @@ def install_scylla_bench(self):
 
     @retrying(n=3)
     def install_cassandra_harry(self):
+        # The SCT code runs from another machine, and the current code runs from the loader machine.
+        # Therefore, one needs to execute the "pwd" command to get the user path.
+        cassandra_harry_folder_path = Path(self.remoter.sudo("pwd", ignore_status=True).stdout.strip())
+        cassandra_harry_path = cassandra_harry_folder_path / "cassandra-harry"
         if self.distro.is_rhel_like:
             self.remoter.sudo("yum install -y git make maven")
         else:
             self.remoter.sudo(shell_script_cmd("""\
                 apt-get update
                 apt-get install -y git make maven
             """))
-        self.remoter.run(shell_script_cmd("""\
-            rm -rf cassandra-harry
-            git clone https://github.com/apache/cassandra-harry
-            cd cassandra-harry
-            git checkout 26bb6696ba1b18ff5c062d780a9360e750691052
-            make mvn
+        # TODO: Needs to uncomment the following lines after https://github.com/apache/cassandra-harry/pull/12 will
```

Comment:
will what?

Reply:
I guess, will be merged.

meanwhile it's working from my fork